server/adapters/prismic.js

```diff
@@ -324,91 +338,107 @@ const getDataForApi = function ( req, listener ) {
         const data = {
             item: null,
             items: null
         };
-        const doQuery = function ( type, uid ) {
+        const doQuery = function (type, uid) {
+            console.log(cache.api);
             let query = [];
-            const navi = getNavi( type );
-            const form = getForm( req, cache.api, type );
-            const isNaviNoForm = (navi && !cache.api.data.forms[ type ]);
-            const done = function ( json ) {
-                if ( !json.results.length ) {
+            const navi = getNavi(type);
+            // const form = getForm(req, cache.api, type);
+            // const isNaviNoForm = (navi && !cache.api.data.forms[type]);
+            const isNaviNoForm = navi && type === "studio";
+            const done = function (json) {
+                console.log(json);
+                if (!json.results.length) {
                     // Static page with no CMS data attached to it...
-                    if ( core.template.cache.pages.indexOf( `${type}.html` ) !== -1 ) {
-                        resolve( data );
+                    if (core.template.cache.pages.indexOf(`${type}.html`) !== -1) {
+                        resolve(data);
 
                     } else {
-                        reject( `${core.config.skylab.name} has no data for the content-type "${type}".` );
+                        reject(`${core.config.skylab.name} has no data for the content-type "${type}".`);
                     }
 
                 } else {
                     // all
                     data.items = json.results;
 
                     // uid
-                    if ( uid || isNaviNoForm ) {
-                        data.item = getDoc( isNaviNoForm ? navi.uid : uid, json.results );
+                    if (uid || isNaviNoForm) {
+                        data.item = getDoc(isNaviNoForm ? navi.uid : uid, json.results);
 
-                        if ( !data.item ) {
-                            reject( `The document with UID "${isNaviNoForm ? navi.uid : uid}" could not be found by ${core.config.skylab.name}.` );
+                        if (!data.item) {
+                            reject(`The document with UID "${isNaviNoForm ? navi.uid : uid}" could not be found by ${core.config.skylab.name}.`);
                         }
                     }
 
-                    resolve( data );
+                    resolve(data);
                 }
             };
-            const fail = function ( error ) {
-                reject( error );
+            const fail = function (error) {
+                reject(error);
             };
 
+            let newType = type;
+
+            if (type === "home") {
+                newType = "project";
+            } else if (type === "studio") {
+                newType = "page";
+            } else if (type === "work") {
+                newType = "project";
+            } else if (type === "play") {
+                newType = "blog";
+            }
```

Comment:
Same thing as above. You could put this into a function and share it in both places.